app/policies/comment_policy.rb

```diff
@@ -0,0 +1,36 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    false
+  end
+  
+  def new?
+    false
+  end
+
+  def edit?
```

Comment:
Great to have a comment policy. 
Reusing the method (update?) makes the code cleaner, 
but I think our default is false since we inherited from application policy, so may not need index, show, new...etc

Reply:
Great point! Thank you Young!